frontend/src/layers/ThQ.ts

```diff
@@ -52,19 +48,32 @@ export class ThQ {
 export const value = (boundaryLayerDepth: number, uWind: number, vWind: number, totalCloudCover: number): number => {
   // Boundary Layer Depth
   const bld = boundaryLayerDepth;
-  const blhCoeff = Math.min(bld / 800, 1); // >800 m = 100%
+  // coeff is 50% for a boundary layer depth of 400 m
+  const bldCoeff = logistic(bld, 400, 4);
 
   // Boundary Layer Wind
   const u = uWind;
   const v = vWind;
   const windForce = Math.sqrt(u * u + v * v);
-  // windForce <= 10 => windCoeff = 1
-  // windForce >  30 => windCoeff ~= 0
-  const windCoeff = Math.exp(-Math.max(windForce - 10, 0) / 10);
+  // coeff is 50% for a wind force of 14 km/h
+  const windCoeff = 1 - logistic(windForce, 15, 6);
```

Comment:
See the shape here: https://www.wolframalpha.com/input?i=plot+1-1%2F%281+%2B+exp%28-%28x-u%29%2Fs%29%29%2C+x%3D0..40%2C+u%3D15%2C+s%3D15%2F6